app/tests/src/wallet/account-manager.js

```diff
@@ -109,8 +82,9 @@ describe('account exploration', () => {
   it('should add consequent new account if previous was used', async () => {
     const mockNet = mockNetwork(ADALITE_CONFIG)
     mockNet.mockBulkAddressSummaryEndpoint()
-    const accountsLength = (await accountManagers[1].discoverAccounts()).length
-    const nextAccount = await accountManagers[1].exploreNextAccount()
+    const accountManager = await accountManagers.withDisabledBulkExport
```

Comment:
I would add to the test description a remark clarifying that this concerns the case when bulk export is disabled as with enabled bulk export this wouldn't make much sense given that all accounts should be discovered on the first go

Reply:
Will do.